src/main/kotlin/no/nav/klage/oppgave/api/view/DokumentView.kt

```diff
@@ -7,15 +7,21 @@ data class DokumenterResponse(val dokumenter: List<DokumentReferanse>, val pageR
 data class DokumentReferanserResponse(val journalpostIder: List<String>)
 
 data class DokumentReferanse(
+    val journalpostId: String,
+    val dokumentInfoId: String?,
```

Comment:
Jeg så egentlig for meg at alle dokumentinfoid'ene lå i listen med Vedlegg. (Men innser at det da ikke kan hete vedlegg..) Jeg er ikke helt sikker på om jeg forstår tankegangen her, og det bærer nok resten av kommentarene mine preg av.. :)